preprocessing/DataCleanser.py

```diff
@@ -1,16 +1,20 @@
 import re
+import numpy as np
 
 
 def dropNulls(df_covid):
     """
-    Removes any rows that contain a null value. We print out the abstract description here because we know there are
-    null value in there. We made the abstract a None type whenever no abstract was present in a JSON file.
+    Removes any rows that contain a null value. We currently create a string 'no abstract provided' if the abstract
+    is not present, so this method is to account for potential mishandled abstract entries.  For 'body_text'
+    entries that are empty strings, we convert them to NaN and then drop them.
 
     Should be completed before running other methods to reduce the number of rows examined.
 
     :param df_covid: Dataframe of covid data.
     :return: Data frame without any null values.
     """
+    df_covid['abstract'].replace(r'^\s+$', 'no abstract provided', inplace=True)
+    df_covid['body_text'].replace(r'^\s+$', np.nan, regex=True)
```

Comment:
Two issues that I see. On line 16, you do not have regex=True
One line 17, you do not have inplace=True